src/app/containers/ATIAnalytics/params/cpsAssetPage/buildParams.js

```diff
@@ -29,10 +30,14 @@ export const buildCpsAssetPageATIParams = (
   const chapter1 = isValidPage ? getChapter1(page) : false;
   const ldpThingIds = getThingAttributes('thingId', pageData);
   const ldpThingLabels = getThingAttributes('thingLabel', pageData);
+  const isLegacyMap = url => url.split('/').length > 7;
```

Comment:
Can we rename this variable to `isLegacyAsset`

Reply:
This is a draft PR so not yet polished for code review

Reply:
Changed it to isLegacyMediaAssetPage to make it clearer; Asset feels vague.

Reply:
This will also be used for STY and PGL pages - how about `isTC2` or `isLegacy`?